olive/passes/onnx/extract_adapters.py

```diff
@@ -280,6 +289,27 @@ def get_sort_key(module_name: str):
 
         return packed_weights, packings
 
+    @staticmethod
+    def save_weights(weights: Dict[str, "NDArray"], output_path: str, save_format: SaveFormat) -> Path:
+        """Save the weights to a file.
```

Comment:
Why not just move this also in common/utils along with load_weights ?

Reply:
Done